src/ontology/mondo-edit.obo

```diff
@@ -619473,6 +619473,26 @@ relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql
 relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/4585 {source="PMID:27683935"} ! GRIN2A
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/9063" xsd:anyURI
 
+[Term]
+id: MONDO:1060146
+name: FGFR1-related Pfeiffer syndrome
+def: "Any pfeiffer syndrome in which the cause of the disease is a mutation in the FGFR1 gene." [https://clinicalgenome.org/affiliation/40059/, PMID:10861678, PMID:14564217, PMID:16957473, PMID:25251565, PMID:7874169]
+synonym: "FGFR1-related Pfeiffer syndrome" EXACT CLINGEN_LABEL []
```

Comment:
This should have the ClinGen group as a x-ref: https://clinicalgenome.org/affiliation/40059/